server/issue.go

```diff
@@ -613,12 +614,52 @@ func (s *Server) createPipelineFromIssue(ctx context.Context, issueCreate *api.I
 			}
 		}
 
+		// Find and use schema from existing tenants.
```

Comment:
I think we need to comment more on this. IIUC, we currently use the schema version of the first database. I am not sure this is the desired behavior or just a temporary method.